.config/nextest.toml

```diff
@@ -1,10 +1,19 @@
 [profile.default]
 slow-timeout = "2m"
-default-filter = 'not test(slow_)'
+default-filter = 'not test(slow_) and not package(tests)'
```

Comment:
Does this mean anyone running `cargo test` will see a failure if they didn't start demo-native?

Reply:
Yes. You need to run `cargo nextest run` now.

Reply:
Or maybe we can address this in cargo configuration. Maybe an alias: https://doc.rust-lang.org/cargo/reference/config.html